Arm/Insts/Cosim/armsimulate

```diff
@@ -1,14 +1,22 @@
+#!/usr/bin/env bash
 # Borrowed from the s2n-bignum project
 # https://github.com/awslabs/s2n-bignum/blob/main/arm/proofs/simulator.c
 # Author(s): John Harrison, Juneyoung Lee
+# Usage: armsimluate <output-file-name> <instruction in GNU assembly syntax>
```

Comment:
```suggestion
# Usage: armsimulate <output-file-name> <32-bit instruction>
```

Reply:
The script doesn't really expect an instruction in assembly syntax, right? It expects a 32-bit value?

Reply:
Also, since you're writing the usage here, might as well talk about all the arguments :)  (copied from `arm_cosim_test` in Arm/Cosim.lean):

```
-- Input args for the armsimulate script:
  --  first, the 32-bit instruction
  --  then 31 64-bit GPRs (no SP)
  --  then 1 4-bit Flags
  --  then 32*2 64-bit SPFs
```

Reply:
Fixed, thanks!